src/model/guild/member.rs

```diff
@@ -535,11 +535,11 @@ impl Display for Member {
 #[non_exhaustive]
 pub struct PartialMember {
     /// Indicator of whether the member can hear in voice channels.
-    pub deaf: bool,
+    pub deaf: Option<bool>,
```

Comment:
This should use serde's `default` attribute, otherwise this is a breaking change that has to go to the next branch.
```suggestion
    #[serde(default)]
    pub deaf: bool,
```